src/views/Questions.vue

```diff
@@ -4,14 +4,14 @@
       <button @click="showQuestions(category)">{{ category.text }}</button>
     </div>
     <div v-if="currentCategory">
-      Soru alani
-      {{ questions[currentCategory] }}
+      <br>Soru alani
```

Comment:
avoid using <br> for styling, go over bootstrap docs and use appropriate class names like `mt-3`, `mb-2`, etc.